fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -449,6 +455,81 @@ private void handleFieldList() throws IOException {
         ctx.getState().setEof();
     }
 
+    // prepared statement cmd COM_EXECUT
+    // protocol
+    // Type             Name Description
+    // int<1>           status [0x17] COM_STMT_EXECUTE
+    // int<4>           statement_id ID of the prepared statement to execute
+    // int<1>           flags  Flags. See enum_cursor_type
+    // int<4>           iteration_count Number of times to execute the statement. Currently always 1.
+    // binary<var>      null_bitmap   NULL bitmap, length= (paramater_count + 7) / 8
+    // int<1>           new_params_bind_flag  Flag if parameters must be re-bound
+    // int<2>           parameter_type  Type of the parameter value. See enum_field_type
+    // string<lenenc>   parameter_name Name of the parameter or empty if not present
+    // binary<var>      parameter_values  value of each parameter
+    // detail https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_stmt_execute.html
+    private void handleExecute() {
+        packetBuf = packetBuf.order(ByteOrder.LITTLE_ENDIAN);
+        // stmt_id
+        int stmtId = packetBuf.getInt();
+        // flag
+        packetBuf.get();
+        packetBuf.getInt();
+        // cache statement
+        PrepareStmtContext prepareCtx = ctx.getPreparedStmt(String.valueOf(stmtId));
+        if (null == prepareCtx) {
+            ctx.getState().setError("msg: Not Found prepared statement, stmtName: " + stmtId);
+            return;
+        }
+        int numParams = prepareCtx.getStmt().getParameters().size();
+        // null bitmap
+        byte[] nullBitmap = new byte[(numParams + 7) / 8];
+        packetBuf.get(nullBitmap);
+        try {
+            ctx.setQueryId(UUIDUtil.genUUID());
+            Integer[] mysqlTypeCodes = new Integer[numParams];
+
+            // new_params_bind_flag
+            if (packetBuf.hasRemaining() && (int) packetBuf.get() != 0) {
+                // parse params types
+                IntStream.range(0, numParams).forEach(i -> mysqlTypeCodes[i] = (int) packetBuf.getChar());
+            }
+            // gene exprs
+            List<Expr> exprs = new ArrayList<>();
+            for (int i = 0; i < numParams; ++i) {
+                if (isNull(nullBitmap, i)) {
+                    exprs.add(new NullLiteral());
+                    continue;
+                }
+                LiteralExpr l = LiteralExpr.parseLiteral(mysqlTypeCodes[i]);
+                l.parseMysqlParam(packetBuf);
+                exprs.add(l);
+            }
+            ExecuteStmt executeStmt = new ExecuteStmt(String.valueOf(stmtId), exprs);
+            // audit will affect performance
+            boolean enableAudit = ctx.getSessionVariable().isAuditExecuteStmt();
+            String originStmt = enableAudit ? executeStmt.toSql() : "/* omit */";
+            executeStmt.setOrigStmt(new OriginStatement(originStmt, 0));
```

Comment:
I suggest to print prepareStmt and parameters separately, or you can print the prepareStmt's name and parameters, it's better than not print

Reply:
currently，  print contents are ExecuteStmt$toSql， only contain  prepareStmt's name and parameters. we have run some tests, found that will influence performance. so here add a config iterm

Reply:
OK, get it,  and I think it's another question? we can split PR to reslove print log performance? because other statements may need to handle too

Reply:
or can you commit a issuse to describe the performance question and your code, it's can help us to optimize it late